plugins/emc/emc_vnx_block_lun_perfdata

```diff
@@ -0,0 +1,505 @@
+ careful. After that, copy .bash_profile from /home/nasadmin to a newly created
+ /home/operator1.
+ 
+ On munin-node side choose a user which will be used to connect through SSH. 
+ Generally user "munin" is ok. Then, execute "sudo su munin -s /bin/bash", 
+ "ssh-keygen" and "ssh-copy-id" to both Control Stations with newly created 
+ user.
+ 
+ Make a link from /usr/share/munin/plugins/emc_vnx_dm_basic_stats to 
+ /etc/munin/plugins/emc_vnx_dm_basic_stats_<NAME>, where <NAME> is any 
+ arbitrary name of your storage system. The plugin will return <NAME> in its 
+ answer as "host_name" field.
+ 
+ For example, assume your storage system is called "VNX5300".
+ Make a configuration file at 
+ /etc/munin/plugin-conf.d/emc_vnx_block_lun_perfdata_VNX5300. For example:
+ 
+ [emc_vnx_block_lun_perfdata_VNX5300]
+ user munin					
+ env.username operator1
+ env.cs_addr 192.168.1.1 192.168.1.2	
+
+ or:
+
+ [emc_vnx_block_lun_perfdata_VNX5300]
+ user munin                                     
+ env.username operator1
+ env.localcli /opt/Navisphere/bin/naviseccli
+ env.sp_addr 192.168.0.3 192.168.0.4
+ env.blockpw foobar
+
+ Where: 
+ user - SSH Client local user
+ env.username - Remote user with Operator role for Block or File part
+ env.cs_addr - Control Stations addresses for remote (indirect) access.
+ env.localcli - Optional. Path of localhost 'Naviseccli' binary. If this 
+   variable is set, env.cs_addr is ignored, and local 'navicli' is used.
+   Requires env.blockpw variable.
+ env.sp_addr - Default is "SPA SPB". In case of "direct" connection to 
+   Storage Processors, their addresses/hostnames are written here.
+ env.blockpw - Password for connecting to Storage Processors
+
+=head1 ERRATA
+
+ It counts Queue Length in not fully correct way. We take parameters totally 
+ from both SP's, but after we divide them independently by load of SPA and SPB.
+ Anyway, in most AAA / ALUA cases the formula is correct.
+
+=head1 HISTORY
+
+ 09.11.2016 - First Release
+ 26.12.2016 - Compatibility with Munin coding style
+
+=cut
+
+export LANG=C
+
+. "$MUNIN_LIBDIR/plugins/plugin.sh"
+
+TARGET=$(echo "${0##*/}" | cut -d _ -f 6)
+
+# "All Storage Processors we have"
+if [[ -v "sp_addr" ]]; then
+	SPALL=$sp_addr
+else
+	SPALL="SPA SPB"
+fi
+# "navicli" command. Can be local or remote, through Control Stations
+if [[ -v "localcli" ]]; then
+	NAVICLI=$localcli
+else
+	NAVICLI="/nas/sbin/navicli"
+fi
+
+
+ssh_check_cmd() {
+		ssh -q $username@$1 "/nasmcd/sbin/getreason | grep -w slot_\`/nasmcd/sbin/t2slot\` | cut -d- -f1"
+}
+
+
+check_conf_and_set_vars () {
+	if [ -z "$username" ]; then
+		echo "No username ('username' environment variable)!"
+		return 1
+	fi
+
+	if [ -z "$localcli" ]; then
+		if [ -z "$cs_addr" ]; then
+			echo "No control station addresses ('cs_addr' environment variable)!"
+			return 1
+		fi
+		#Choosing Cotrol Station. Code have to be "10"
+		for CS in $cs_addr; do
+			if [[ "10" -eq "$(ssh_check_cmd $CS)" ]]; then
```

Comment:
Okay, now i understood you. Yes, i have "10 " in the output with trailing space:
``` ssh_check_cmd $CS | hd
00000000  31 30 20 0a                                       |10 .|
```
But anyway, "10" - is not a number! It's a piece of string, grepped from output of `/nasmcd/sbin/getreason`. Why do i have to compare it arithmetically? :)
Anyway, if you insist to make it without space an use '=', i'll do that.